src/EnergyPlus/HighTempRadiantSystem.cc

```diff
@@ -950,16 +950,16 @@ namespace HighTempRadiantSystem {
                     HeatBalanceSurfaceManager::CalcHeatBalanceInsideSurf(state, ZoneNum);
 
                     // Redetermine the current value of the controlling temperature
-                    auto &thisZoneHB = state.dataZoneTempPredictorCorrector->zoneHeatBalance(ZoneNum);
+                    auto &thisZoneHBMod = state.dataZoneTempPredictorCorrector->zoneHeatBalance(ZoneNum);
                     switch (thisHTR.ControlType) {
                     case RadControlType::MATControl: {
-                        ZoneTemp = thisZoneHB.MAT;
+                        ZoneTemp = thisZoneHBMod.MAT;
                     } break;
                     case RadControlType::MRTControl: {
-                        ZoneTemp = thisZoneHB.MRT;
+                        ZoneTemp = thisZoneHBMod.MRT;
                     } break;
                     case RadControlType::OperativeControl: {
-                        ZoneTemp = 0.5 * (thisZoneHB.MAT + thisZoneHB.MRT);
+                        ZoneTemp = 0.5 * (thisZoneHBMod.MAT + thisZoneHBMod.MRT);
```

Comment:
[src/EnergyPlus/HighTempRadiantSystem.cc:953]:(style),[shadowVariable],Local variable 'thisZoneHB' shadows outer variable
[src/EnergyPlus/HighTempRadiantSystem.cc:814]:(style),[constVariable],Variable 'thisZoneHB' can be declared as reference to const
[src/EnergyPlus/HighTempRadiantSystem.cc:907]:(style),[constVariable],Variable 'thisZoneHB' can be declared as reference to const
[src/EnergyPlus/HighTempRadiantSystem.cc:953]:(style),[constVariable],Variable 'thisZoneHB' can be declared as reference to const